src/source_map_support.js

```diff
@@ -0,0 +1,112 @@
+var vlqMap = {};
+'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789+/='.split('').forEach(function (c, i) {
+  vlqMap[c] = i;
+});
+
+// based on https://github.com/Rich-Harris/vlq/blob/master/src/vlq.ts
+function decodeVLQ(string) {
```

Comment:
it would be good to have as few as possible things at the top level (better for minification). can these be moved into the place that uses them?